### PR TITLE
Adding button add-in and tile config examples to template

### PR DESCRIPTION
**Pull request by Blackbaud-ChristopherRodgers**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   "license": "ISC",
   "dependencies": {
     "@blackbaud/skyux": "2.25.0",
-    "@blackbaud/sky-addin-client": "^1.0.2",
-    "@blackbaud/skyux-lib-addin-client": "^1.1.2"
+    "@blackbaud/sky-addin-client": "1.0.8",
+    "@blackbaud/skyux-lib-addin-client": "^1.1.3"
```
Comment (michael-tims) [inline package.json]: Should `^` be removed here as well?

File: src/app/shared/components/my-tile-settings/my-tile-settings-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public context: MyTileSettingsContext,
+    public instance: SkyModalInstance,
+    private waitSvc: SkyWaitService
+  ) {
+  }
+
+  public save() {
+    this.waitSvc.beginBlockingPageWait();
+
+    // simulate performing a save operation that takes 1 seconds
```
Comment (michael-tims) [inline src/app/shared/components/my-tile-settings/my-tile-settings-modal.component.ts]: seconds -> second

File: src/app/shared/components/my-tile-settings/my-tile-settings.component.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { SkyModalService, SkyModalCloseArgs } from '@blackbaud/skyux/dist/core';
+import { AddinClientService } from '@blackbaud/skyux-lib-addin-client';
+import { AddinClientInitArgs } from '@blackbaud/sky-addin-client';
+import { MyTileSettingsContext } from './my-tile-settings-context';
+import { MyTileSettingsModalComponent } from './my-tile-settings-modal.component';
+
+@Component({
+    selector: 'my-tile-settings',
+    template: '',
+    styleUrls: ['./my-tile-settings.component.scss']
```
Comment (michael-tims) [inline src/app/shared/components/my-tile-settings/my-tile-settings.component.ts]: Remove extra spacing

**Review by michael-tims — COMMENTED** [review]
Looks good.  Just a few minor things.

**Review by michael-tims — APPROVED** [review]

**Review by Blackbaud-AdamHickey — APPROVED** [review]

**Pull request merged**